### DISCUSSION TITLE
Electromagnetics Module 3D capability

### DISCUSSION THREAD

**Discussion by nathmay** [idaholab/moose#27162] [Q&A Modules: Electromagnetics]
I noticed that the [Electromagnetics Module documentation](https://mooseframework.inl.gov/modules/electromagnetics/index.html) says:
> ... (3D is currently in development)

I was curious if there are any existing branches where this is being tested, and if there are tentative plans for release dates!

**Comment by GiudGiud** [comment]
@cticenhour
